src/Utility.ts

```diff
@@ -87,11 +87,13 @@ export namespace Utility {
         if (config) {
             // tslint:disable-next-line:no-backbone-get-set-outside-model
             const workspace: string = config.get<string>('workspace');
-            try {
-                await fse.ensureDir(workspace);
-                return workspace;
-            } catch (err) {
-                return path.join(defaultStoragePath, 'tomcat');
+            if (!workspace.startsWith('<<')) {
```

Comment:
Just a question: what does '<<' mean here?

Reply:
default workspace settings: "default": "<<fill in tomcat servers workspace path>>",

Reply:
I see. Actually, I prefer to leave the default value as empty. 

Leave the choice to you